source/Sashimi.Tests.Shared/Server/ActionHandlerTestBuilder.cs

```diff
@@ -92,7 +92,7 @@ public ActionHandlerTestBuilder<TCalamariProgram> WithAssert(Action<TestActionHa
             return this;
         }
 
-        public TestActionHandlerResult Execute(bool assertWasSuccess = true)
+        public TestActionHandlerResult Execute(bool assertWasSuccess = true, bool runInProc = true)
```

Comment:
It'll be clearer if you call this `forceRunOutProc`

Reply:
I'll fix this in the other PR

Reply:
Actually the change is done